pygfunction/heat_transfer.py

```diff
@@ -295,12 +291,14 @@ def similarities(boreholes, splitRealAndImage=True, disTol=0.1, tol=1.0e-6,
         Set to True if similarities are evaluated separately for real and image
         sources. Set to False if similarities are evaluated for the sum of the
         real and image sources.
-    distol : float, defaults to 0.1
+    disTol : float, defaults to 0.1
         Absolute tolerance (in meters) on radial distance. Two distances
         (d1, d2) between two pairs of boreholes are considered equal if the
         difference between the two distances (abs(d1-d2)) is below tolerance.
+
+    # tol is not used
```

Comment:
This isn't passed through to anything.

Reply:
I've found the missing argument in calls to `_similarities_one_distance`.